crates/revm/src/context/context_precompiles.rs

```diff
@@ -144,29 +130,28 @@ impl<DB: Database> ContextPrecompiles<DB> {
     /// Clones the precompiles map if it is shared.
     #[inline]
     pub fn to_mut(&mut self) -> &mut HashMap<Address, ContextPrecompile<DB>> {
-        match self.inner {
-            PrecompilesCow::Default(_) => self.to_mut_slow(),
-            PrecompilesCow::Owned(ref mut inner) => inner,
-        }
-    }
+        self.mutate_into_owned();
 
-    #[allow(clippy::wrong_self_convention)]
-    #[cold]
-    fn to_mut_slow(&mut self) -> &mut HashMap<Address, ContextPrecompile<DB>> {
-        let PrecompilesCow::Default(precompiles) = self.inner else {
+        let PrecompilesCow::Owned(inner) = &mut self.inner else {
             unreachable!()
         };
+        inner
+    }
+
+    /// Mutates Self into Owned variant, or do nothing if it is already Owned.
+    /// Mutation will clone all precompiles.
```

Comment:
This is what `to_mut` does, this does not need to be public

Reply:
True, will make it private